src/eos_gates.js

```diff
@@ -98,20 +186,26 @@ const Application = new Lang.Class({
                                 use_markup: true,
                                 wrap: true,
                                 max_width_chars: 30,
-                                label: _("You can install applications from our <a href='endlessm-app://eos-app-store'>App Store</a>.") });
+                                label: this.getHelpMessage() });
         label.get_style_context().add_class('unsupported-subtitle');
         label.connect('activate-link', Lang.bind(this, function() {
             this.quit();
             return false;
         }));
         errorMessageBox.add(label);
 
-        box.add(errorMessageBox);
+        let extraInformationMessage = this.getExtraInformationMessage();
+        if (extraInformationMessage) {
+            label = new Gtk.Label({ visible: true,
+                                    use_markup: true,
+                                    wrap: true,
+                                    max_width_chars: 30,
+                                    label: this.getExtraInformationMessage() });
```

Comment:
Use `extraInformationMessage` instead of calling `this.getExtraInformationMessage()` again here.